common/test/acceptance/tests/studio/test_studio_components.py

```diff
@@ -0,0 +1,69 @@
+"""
+Acceptance tests for adding components in Studio.
+"""
+import ddt
+
+from .base_studio_test import ContainerBase
+from ...fixtures.course import XBlockFixtureDesc
+from ...pages.studio.container import ContainerPage
+from ...pages.studio.utils import add_component
+
+
+@ddt.ddt
+class AdvancedProblemComponentTest(ContainerBase):
+    """
+    Feature: CMS.Component Adding
+    As a course author, I want to be able to add a wide variety of components
+    """
+    __test__ = True
```

Comment:
This signals to nose discovery that this is indeed a test to execute. 

That should be signaled already by the filename and classname matching the conventions, but for some reason that wasn't the case in some other tests so someone stuck this in one of our bokchoy files and the convention spread. 

I'm actually not sure myself if it's required. I'll experiment and leave it out if it is not.